README.md

```diff
@@ -145,6 +145,9 @@ protect hosts in whatever other ways you see fit.
     * [Zentral](https://github.com/zentralopensource/zentral/wiki) - A
       centralized service that pulls data from multiple sources and deploy
       configurations to multiple services.
+    * [Zercurity](https://github.com/zercurity/zercurity) - A dockerized
+      service combining Santa + Osquery to manage and monitor applications
```

Comment:
This reads a little unclearly to me - it sounds like it runs Santa and Osquery inside docker. Perhaps `A dockerized service for managing and monitoring applications across a large fleet utilizing Santa + Osquery`?

Reply:
Hi, Russel. Happy with your suggestion. I'll get that changed. Zercurity is closed source (we've had support for Google Santa for around 2.5 years now). The docker version is a community version which works completely. Aside from official commercial support and we don't provide all of the pre-configured Osquery packs for compliance etc. You have to bring your own. Having said that all the Google Santa works out of the box there is no limit on this. I'm happy to have a call to go over the project a bit more or how our sync service works?

Reply:
That's OK, I was only asking because if it _were_ open source I could spot-check the sync code for you. But if it's not I'll take your word for it 😄